aries-site/src/layouts/main/Layout.js

```diff
@@ -69,15 +63,10 @@ export const Layout = ({
                 }}
               >
                 {children}
-                {!isLanding && !isNavPage && (
-                  <>
-                    <FeedbackSection />
-                    <NextContent color={color} nextContent={nextContent} />
-                  </>
-                )}
+                {!isLanding && !isNavPage && <FeedbackSection />}
```

Comment:
Is `isNavPage` used any where any more? If not, can we remove it from this file altogether?

Reply:
You're right, we're good to remove this now.